resources/property.py

```diff
@@ -77,7 +77,12 @@ def get(self, id):
         rentalProperty = PropertyModel.find_by_id(id)
 
         if rentalProperty:
-            return rentalProperty.json()
+            json = rentalProperty.json()
+            json["tenants"] = []
+            for lease in rentalProperty.leases:
+                json["tenants"].append(lease.tenant.json())
+            return json
+
```

Comment:
This is a stylistic thing, but I think we should probably avoid using `json` as a variable name, since it's already used as a library, method and kwarg throughout the app. Perhaps change this to `property_data` or `property_json`?

Reply:
Great, renaming to `property` given Nick's and your suggestion! Thank you, and good catch!